lib/supa/builder.rb

```diff
@@ -21,6 +21,12 @@ def initialize(subject, representer:, tree:)
       end
     end
 
+    def attributes(names, options = {})
```

Comment:
1) Tbh, I don't like exceptions of rules as people tend to add more and more if there's at least one already. We could refactor the code this way:
```ruby
COMMANDS.each do |command|
  klass = Supa::Commands.const_get(command.capitalize)

  define_method(command) do |*args, &block|
    options = args[-1].is_a?(Hash) ? args.pop : {}

    klass.new(@subject,
      representer: @representer,
      tree: @tree,
      args: args,
      options: options,
      &block).represent
  end
end
```

2) I'm not convinced if we really need a new command, the same can be achieved by just calling `attribute` for every single argument of `attributes`, hence you don't have to implement the same functionaliy twice as well as break the interface of commands.
```ruby
def attributes(*args)
  options = args[-1].is_a?(Hash) ? args.pop : {}
  names = args

  names.each do |name|
    attribute(name, options)
  end
end
```